City-of-Helsinki/drupal-infofinland
# UHF-10572 contact form

# [UHF-10572](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10572)
Prevent contact form from saving data to database

## What was done
Update hook to remove all webform submissions from database
Applied a patch which fixes the 500 error when the DB saving has been disabled.
Changed the custom email hander to not to be saved in database

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10572`
  * `make fresh`
* Run `make drush-cr`

## How to test
Go and see that webform submissions are deleted by the update hook (admin/structure/webform/submissions/manage)

**Email testing from [UHF-10025](https://github.com/City-of-Helsinki/drupal-infofinland/pull/304)**
- Go to Turku-page since it has different feedback email address set to node
- Add breakpoint to EmailWebformHandler::sendMessage line 1167 and submitted feedback from frontend.
  - The email address should be something else than the default infofinland-address 


[UHF-10572]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10572?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[UHF-10025]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10025?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
